cetmix_tower_server/security/cx_tower_variable_value_security.xml

```diff
@@ -1,16 +1,82 @@
         <field name="domain_force">['|', ('is_global', '=', True),
             ('server_id.message_partner_ids', 'in', [user.partner_id.id])]</field>
         <field
             name="groups"
             eval="[(4, ref('cetmix_tower_server.group_user')), (4, ref('cetmix_tower_server.group_manager'))]"
         />
+    </record>
+
+    <record id="cx_tower_variable_value_action_rule_group_user_access" model="ir.rule">
+        <field
+            name="name"
+        >Tower variable value: user access to variable values in plan line action
+            rule</field>
+        <field name="model_id" ref="model_cx_tower_variable_value" />
+        <field name="domain_force">
+            [
+            ('plan_line_action_id.access_level', '=', '1'),
+            '|',
+            ('plan_line_action_id.line_id.plan_id.server_ids', '=', False),
+            ('plan_line_action_id.line_id.plan_id.server_ids.message_partner_ids', 'in', [user.partner_id.id])
+            ]
+        </field>
+        <field name="groups" eval="[(4, ref('cetmix_tower_server.group_user'))]" />
+    </record>
 
+    <record
+        id="cx_tower_variable_value_action_rule_group_manager_access"
+        model="ir.rule"
+    >
+        <field
+            name="name"
+        >Tower variable value: manager access to variable values in plan line
+            action rule</field>
+        <field name="model_id" ref="model_cx_tower_variable_value" />
+        <field name="domain_force">
+            [
+            ('plan_line_action_id.access_level', '&lt;=', '2'),
+            '|',
+            ('plan_line_action_id.line_id.plan_id.server_ids', '=', False),
+            ('plan_line_action_id.line_id.plan_id.server_ids.message_partner_ids', 'in', [user.partner_id.id])
+            ]
+        </field>
+        <field name="groups" eval="[(4, ref('cetmix_tower_server.group_manager'))]" />
+        <field name="perm_read" eval="1" />
+        <field name="perm_create" eval="1" />
+        <field name="perm_write" eval="1" />
+        <field name="perm_unlink" eval="0" />
+    </record>
+
+    <record
+        id="cx_tower_variable_value_action_rule_group_manager_unlink"
+        model="ir.rule"
+    >
+        <field
+            name="name"
+        >Tower variable value: manager delete own variable values in plan line
+            action</field>
+        <field name="model_id" ref="model_cx_tower_variable_value" />
+        <field name="domain_force">
+            [
+            ('plan_line_action_id.create_uid', '=', user.id),
+            ('plan_line_action_id.access_level', '&lt;=', '2'),
+            '|',
+            ('plan_line_action_id.line_id.plan_id.server_ids', '=', False),
+            ('plan_line_action_id.line_id.plan_id.server_ids.message_partner_ids', 'in', [user.partner_id.id])
+            ]
+        </field>
+        <field name="groups" eval="[(4, ref('cetmix_tower_server.group_manager'))]" />
+        <field name="perm_unlink" eval="1" />
```

Comment:
"1" is the default value for the "perm_..." fields. Which means you need to set it to "0" explicitly. Otherwise this rule is applied to all other actions too.
NB: this case should have been caught by tests, looks like they should be fixed or updated.

Reply:
fixed